assets/components/paymentButtons/directDebitPopUpButton/directDebitPopUpButton.jsx

```diff
@@ -0,0 +1,81 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { connect } from 'react-redux';
+import type { Currency } from 'helpers/internationalisation/currency';
+import * as storage from 'helpers/storage';
+import { openDirectDebitPopUp } from 'components/directDebit/directDebitActions';
+import DirectDebitPopUpForm from 'components/directDebit/directDebitPopUpForm/directDebitPopUpForm';
+
+// ---- Types ----- //
+
+/* eslint-disable react/no-unused-prop-types */
+type PropTypes = {
+  amount: number,
+  callback: Function,
+  currency: Currency,
+  email: string,
+  isTestUser: boolean,
+  isPopUpOpen: boolean,
+  openDirectDebitPopUp: () => void,
+};
+/* eslint-enable react/no-unused-prop-types */
+
+// ----- Map State/Props ----- //
+
+function mapStateToProps(state) {
+  return {
+    isPopUpOpen: state.page.directDebit.isPopUpOpen,
```

Comment:
This is a component, does it make sense to have a `page` scope?

Reply:
As it was offline discussed, this is an page component.